src/fmt.rs

```diff
@@ -91,6 +80,16 @@ impl Config {
             key_value_newlines: true,
             allowed_blank_lines: 1,
             crlf: false,
+            table_order: [
+                "package",
+                "features",
+                "dependencies",
+                "build-dependencies",
+                "dev-dependencies",
+            ]
```

Comment:
Will this order be enforced by default? I don't understand exactly what the command line flag would be for using this (as opposed to defining a custom ordering, which to my understanding is `-o foo,bar,baz`).

Reply:
No the `Config::new()` is only used by the fuzzer and in a few tests. I should probably make that more clear by feature gating it. I added 
```rust
#[cfg(any(test, feature = "fuzz"))]
pub(crate) fn new() -> Self {
```